tests/e2e/features/artifacts.feature

```diff
@@ -269,4 +269,67 @@ Feature: Artifacts - UI
+    Then Element with "href and text" "'https://era.zksync.io/docs/dev/tutorials/custom-aa-tutorial.html' and 'Account abstraction'" should be "clickable"
+    Then Element with "href and text" "'https://era.zksync.io/docs/dev/tutorials/custom-paymaster-tutorial.html' and 'Building custom Paymasters'" should be "visible"
+    Then Element with "href and text" "'https://era.zksync.io/docs/dev/tutorials/custom-paymaster-tutorial.html' and 'Building custom Paymasters'" should be "clickable"
+    Then Element with "href and text" "'https://era.zksync.io/docs/dev/tutorials/cross-chain-tutorial.html' and 'Cross-chain governance'" should be "visible"
+    Then Element with "href and text" "'https://era.zksync.io/docs/dev/tutorials/cross-chain-tutorial.html' and 'Cross-chain governance'" should be "clickable"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/js' and 'Javascript SDK'" should be "visible"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/js' and 'Javascript SDK'" should be "clickable"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/hardhat' and 'Hardhat plugins'" should be "visible"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/hardhat' and 'Hardhat plugins'" should be "clickable"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/tools/zksync-cli/' and 'zkSync Era CLI'" should be "visible"
+    Then Element with "href and text" "'https://era.zksync.io/docs/api/tools/zksync-cli/' and 'zkSync Era CLI'" should be "clickable"
+    Then Element with "partial text" "Guides" should be "visible"
+    Then Element with "partial text" "Tools" should be "visible"
+
+  @id1529 @upperNavigationMenu
+  Scenario: Check artifacts on the Upper navigation menu (Learn)
+    Given I am on the Main page
+    When I hover the "text" element with " Learn " value
+    Then Element with "href and text" "'https://zksync.io/ethos' and 'Freedom is our mission'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/ethos' and 'Freedom is our mission'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/hyperscalability' and 'Hyperscalibility'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/hyperscalability' and 'Hyperscalibility'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/security' and 'Security'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/security' and 'Security'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/ux' and 'Game-changing UX'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/ux' and 'Game-changing UX'" should be "clickable"
+
+  @id1531 @upperNavigationMenu
+  Scenario: Check artifacts on the Upper navigation menu (Network)
```

Comment:
based on id1530 here we can also add: 
Then Element with "partial text" "<Text>" should be "visible"

ZKSYNC ERA (V2)
ZKSYNC LITE (V1)
ECOSYSTEM